content/gl/code-of-conduct/index.md

```diff
@@ -79,21 +79,28 @@ eliminar, editar ou rexeitar mensaxes, comentarios, commits, código, edicións
 tickets e outras contribucións que non se axusten a este código de conduta, e comunicará os motivos
 para as suas decisións de moderación cando proceda.
 
-## Información de queixa e contacto
+## Protocolo de queixas e denuncias
 
 Os casos de comportamento abusivo, de acoso ou doutro xeito inaceptable pódense informar aos
-xestores da comunidade responsables do cumprimento:
+xestores da comunidade responsables do cumprimento.
+
+Se sintes que algún dos citados comportamentos tivo lugar ou aprezas que alguén cometeu un acto de ofensa, te animamos a tomar o seguinte curso de acción tan axiña como sexa posible:
```

Comment:
```suggestion
Se sintes que algún dos citados comportamentos tivo lugar ou aprezas que alguén cometeu un acto de ofensa, animamoste a tomar o seguinte curso de acción tan axiña como sexa posible:
```
Moito ollo cos pronomes
https://youtu.be/HfAb4xH1dD8?si=Vb5r5ddzyPHrEB91
https://youtu.be/rdaCfWWvQp4?si=9ecUn5xfw-XwEI_x